src/js/controller.js

```diff
@@ -0,0 +1,23 @@
+export default class Controller {
+    constructor(model, view){
+        this.model = model;
+        this.view = view;
+    }
+
+    init(view) {
+        this.view = view;
```

Comment:
Do not repeat yourself DRY. Bu classda artiq var view, onu təkrar burda niyə ötürürsən